plugins/utils/alpha.nix

```diff
@@ -24,7 +24,7 @@ with lib; let
 
       val = mkOption {
```

Comment:
```suggestion
      val = helpers.mkNullOrOption
        (with helpers.nixvimTypes;
          oneOf [
            str
            int
            (listOf (either str (attrsOf anything)))
          ]
        )
        "Value for section."
```

Reply:
This is a suggestion, I am not agains the `mkOption`.

Reply:
I hesitated, but I feel like the mkNullOprOption would be more consistent